### PR TITLE
Remove html tables from readme template markdown

### PR DESCRIPTION
**Pull request by krazik**
### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### Description
Update readme files to remove html template 
https://github.com/fastlane/fastlane/issues/11196

### CODE REVIEW

**Comment by krazik** [comment]
I signed it! 

tho it's just a damn readme ;) lol

File: fastlane/lib/fastlane/documentation/docs_generator.rb [hunk 1]
```diff
@@ -16,23 +16,10 @@ def self.run(ff, output_path = nil)
...
-      output << "<td width=\"33%\" align=\"center\">macOS</td>"
-      output << "<td width=\"33%\" align=\"center\">macOS or Linux with Ruby 2.0.0 or above</td>"
-      output << "</tr>"
-      output << "<tr>"
-      output << "<td width=\"33%\"><code>brew cask install fastlane</code></td>"
-      output << "<td width=\"33%\"><a href=\"https://download.fastlane.tools\">Download the zip file</a>. Then double click on the <code>install</code> script (or run it in a terminal window).</td>"
-      output << "<td width=\"33%\"><code>sudo gem install fastlane -NV</code></td>"
-      output << "</tr>"
-      output << "</table>"
+      output << "| Homebrew                    | Installer Script                                | RubyGems                                |"
```
Comment (janpio) [inline fastlane/lib/fastlane/documentation/docs_generator.rb]: missing link for Homebrew now

File: fastlane/lib/fastlane/documentation/docs_generator.rb [hunk 2]
```diff
@@ -16,23 +16,10 @@ def self.run(ff, output_path = nil)
...
+      output << "|-----------------------------|-------------------------------------------------|-----------------------------------------|"
+      output << "| macOS                       | macOS                                           | macOS or Linux with Ruby 2.0.0 or above |"
+      output << "| `brew cask install fastlane`| [Download zip](https://download.fastlane.tools) | `sudo gem install fastlane -NV`         |"
```
Comment (janpio) [inline fastlane/lib/fastlane/documentation/docs_generator.rb]: Installer script instructions changed

File: match/lib/assets/READMETemplate.md [hunk 3]
```diff
@@ -16,23 +16,11 @@ xcode-select --install
 
 ### Choose your installation method:
```
Comment (janpio) [inline match/lib/assets/READMETemplate.md]: see above

**Comment by krazik** [comment]
updated

**Comment by krazik** [comment]
updated

**Comment by krazik** [comment]
one build failed because of circleci infra problems but I can't retrigger

**Comment by janpio** [comment]
Seems to have fixed itself.

Any reason why it would not be a good idea to switch this html table to markdown @KrauseFx?

**Comment by janpio** [comment]
The `33%` with goes missing with this conversion.

Screenshot from Bitbucket:
image

Comparison on Github:
image

**Comment by janpio** [comment]
Seems not to.

But we might switch the orientation of the table:
image

Code:
```
| Method                     | OS support                              | Description                                                                                                                           |
|----------------------------|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
| [Homebrew](http://brew.sh) | macOS                                   | `brew cask install fastlane`                                                                                                          |
| Installer&nbsp;Script      | macOS                                   | [Download the zip file](https://download.fastlane.tools). Then double click on the `install` script (or run it in a terminal window). |
| RubyGems                   | macOS or Linux with Ruby 2.0.0 or above | `sudo gem install fastlane -NV`                                                                                                       | 
```

What do you think @krazik?

This might be especially useful if/when we add other installation methods or platforms to be supported...

**Comment by krazik** [comment]
I don't hav any opinion on orientation, I wast just trying to match the old one.

**Comment by janpio** [comment]
We talked about it internally, and changing the orientation would be a great thing. 
Could you change your branch? Or should I just jump in and commit the changes?

**Comment by krazik** [comment]
updated

**Review by janpio — APPROVED** [review]
🥇

**Comment by janpio** [comment]
(Those non breaking spaces are sneaky bastards... my 2 commits were to make sure they are actually there)

**Comment by krazik** [comment]
cla confirm

**Comment by janpio** [comment]
Yeah, nothing you have to do about that. It is because I committed as well, then @googlebot switches to manual mode. 

I will merge this as soon as the tests are green.

Thanks for your work!

This will also make its way over to the documentation with https://github.com/fastlane/docs/pull/502

**Pull request merged**